packages/manager/apps/dedicated/client/app/components/ip/agoraOrder/translations/Messages_fr_FR.json

```
@@ -1,6 +1,19 @@
 {
-  "ip_agora_table_header_offer": "Offre",
+  "ip_agora_table_header_offer": "Choisissez votre offre",
   "ip_agora_table_header_quantity": "Quantité",
   "ip_agora_table_header_country": "Localisée(s) en",
-  "ip_agora_table_header_organisation": "Organisation"
+  "ip_agora_table_header_organisation": "Associer une organisation",
+  "ip_agora_failover_ip_title": "Additional IP",
+  "ip_agora_block_ip_title": "Bloc Additional IP",
+  "ip_agora_select_quantity_title": "Choisissez votre quantité",
+  "ip_agora_ip_localisation_title": "IP localisation",
+  "ipv4_select_placeholder": "Selectionner une taille",
+  "license_dashboard_title": "Commander une IP",
+  "ipv4Failover_price_label": "1,50 HT/mois",
+  "ipv4Block_description": "Bloc d'adresse IPv4. Les tailles de bloc disponibles varient en fonction du service associé.",
+  "ipv4Block_price_label": "1,50 HT/mois par adresse IP",
+  "ipv4Block_organization_description": "Vous devez associer une organisation à votre bloc d'adresses IP. L'organisation représente l'entité administrative détentrice du bloc d'adresses IP et doit obligatoirement être déclarée dans la base de données publique Whois (registre mondial des adresses IP et noms de domaines). L'organisation contient les informations de contact suivants : nom,  type de RIR, et adresse email. <a class=\"oui-link oui-link_icon\" href=\"{{ url }}\">En savoir plus sur les organisations.<span class=\"oui-icon oui-icon-external-link\" aria-hidden=\"true\"></span></a> Si vous n'associez pas d'organisation, les informations de votre compte OVHcloud (NIC) pourront être utilisées",
```

i guess we can do this, it will avoid us the HTML into content.
```suggestion
  "ipv4Block_organization_description": "Vous devez associer une organisation à votre bloc d'adresses IP. L'organisation représente l'entité administrative détentrice du bloc d'adresses IP et doit obligatoirement être déclarée dans la base de données publique Whois (registre mondial des adresses IP et noms de domaines). L'organisation contient les informations de contact suivants : nom,  type de RIR, et adresse email. Si vous n'associez pas d'organisation, les informations de votre compte OVHcloud (NIC) pourront être utilisées",
  "ipv4Block_organization_description_know_more": "En savoir plus sur les organisations."
```